modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -56,6 +56,19 @@ def create_session?
     end
   end
 
+  def run
+    results = super
+    logins = results.flat_map { |_k, v| v[:successful_logins] }
+    sessions = results.flat_map { |_k, v| v[:successful_sessions] }
+    print_status("Bruteforce completed, #{logins.size} credentials were successful.")
+    if datastore['CreateSession']
+      print_status("#{sessions.size} MSSQL sessions were opened successfully.")
```

Comment:
```suggestion
      print_status("#{sessions.size} MSSQL #{sessions.size == 1 'session was' ? 'sessions were'} opened successfully.")
```

(And for the others)